sql/2020/17_JAMstack/ssg_compared_to_2019.sql

```diff
@@ -0,0 +1,58 @@
+#standardSQL
+# SSG adoptions and top SSGs YoY
+SELECT
+  _TABLE_SUFFIX AS client,
+  2020 AS year,
+	app AS ssg,
+  COUNT(0) AS freq,
+  total,
+  COUNT(0) / total AS pct
+FROM
+  `httparchive.technologies.2020_08_01_*`
+JOIN (
+  SELECT
+    _TABLE_SUFFIX,
+    COUNT(0) AS total
+  FROM
+    `httparchive.summary_pages.2020_08_01_*`
+  GROUP BY
+    _TABLE_SUFFIX)
+USING
+  (_TABLE_SUFFIX)
+WHERE
+  LOWER(category) = "static site generator" OR
+  app = "Next.js"
+GROUP BY
+  client,
+  total,
+  ssg
+UNION ALL
+SELECT
+  _TABLE_SUFFIX AS client,
+  2019 AS year,
+	app AS ssg,
+  COUNT(0) AS freq,
+  total,
+  COUNT(0) / total AS pct
+FROM
+  `httparchive.technologies.2019_07_01_*`
+JOIN (
+  SELECT
+    _TABLE_SUFFIX,
+    COUNT(0) AS total
+  FROM
+    `httparchive.summary_pages.2019_07_01_*`
+  GROUP BY
+    _TABLE_SUFFIX)
+USING
+  (_TABLE_SUFFIX)
+WHERE
+  LOWER(category) = "static site generator" OR
+  app = "Next.js"
```

Comment:
Aside: should Next.js be categorized as SSG? If so could you open an issue on the Wappalyzer repo? https://github.com/AliasIO/wappalyzer/

Reply:
For now it's hard to say. I included that regarding our last discussion https://github.com/HTTPArchive/almanac.httparchive.org/pull/1228#issuecomment-686729279
I expect some reaction from Next.js team after publishing the chapter, so they can add some markers which will help Wappalyzer detect Next.js when it's used as SSG.